src/Directory.Build.props

```diff
@@ -12,8 +12,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <ZebusVersion>3.5.0</ZebusVersion>
-    <ZebusContractsVersion>3.0.0</ZebusContractsVersion>
+    <ZebusVersion>3.5.1</ZebusVersion>
```

Comment:
New feature = new minor version, but it would be better *not* to change the version number in a branch.